### ISSUE TITLE
Adding Non-Cluster Hosts

### ISSUE THREAD

**Issue by ilisei** [canonical/lxd-ui#632]
Good afternoon, as I understand at the moment it is not possible to add a separate lxc host to the control panel. To manage two at once. Is such functionality planned?

**Comment by edlerd** [comment]
Hello,

if I understand you correctly, you want to switch between different LXD backends from one LXD UI. The backends are distinct, so not in one cluster. Is this correct?

We indeed have such a feature on our radar. It is only planned in the far future, more towards mid/end of this year.

You can help us by describing your use case and how you would like it to work to satisfy your needs. I have thought about this a bit, but some moving parts are still unclear to me. Especially how the configuration (adding/editing/removing) of distinct servers would work and where we would store this config. I think conceptually distinct servers and distinct clusers would be the same.

**Comment by ilisei** [comment]
Yes, you understood everything correctly. The idea is that there are several hosts with different configurations, I would like to manage them from one console. As an example, I saw a similar implementation in the project https://github.com/turtle0x1/LxdMosaic
As an example, you can describe the vmware management console. That is, I have direct visibility through the network between hosts.
